### ISSUE TITLE
Multiselect - unwanted tag creation when typing

### ISSUE THREAD

**Issue by AnitaMihai** [telerik/kendo-angular#1767]
## I'm submitting a...

* Bug report 

## Current behavior

When typing into the multiselect control in order to filter the collection of items unwanted tags are created. If we have available 3 items: "Small", "Small1", "Small2" and we filter the list (writing "s" for example) and select the "Small" element, when trying to filter the second time the list using the same letter "s", a new tag is created with the "Small" tag, which shouldn't happen. 

## Expected behavior

When filtering the tags shouldn't be created unless we select something from the filtered list.
## Minimal reproduction of the problem with instructions

In order to reproduce the issue we can use the forked example that I created from the Multiselect  control, the basic Filtering section: (I just added a few more items on the source collection)

https://stackblitz.com/edit/angular-b2jaeq

and follow the steps:
1. Write "S" on the box and select the first element , being "Small"
2. Write "S" again on the box ,wait a little bit and now it can be seen a new tag added with the same name "Small" but that isn't anything that was selected from the list, Also when selecting nothing from the suggested list and just click outside the box (in order to lost the focus) it can be seen that the "Small" tag it is added there on the list and upon some time it just disappears.

## Environment

Package versions:

```
Package versions: the newest version of Angular and kendo-dropdowns packages.
```

Browser:

- Chrome (desktop) version 68

**Comment by vbakavets** [comment]
We faced the same issue today. Any fix or workaround would be helpful.

**Comment by patch321** [comment]
Did this ever get patched? I'm using dropdowns 3.1 and still experiencing this problem.

**Comment by dimitar-pechev** [comment]
The fix is available in the [development build](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/) of the kendo-angular-dropdowns package v3.2.0-dev.201810111406.

**Comment by dimitar-pechev** [comment]
Shipped in v3.2.0 of the `@progress/kendo-angular-dropdowns` package.
